gl_rubocop.gemspec

```diff
@@ -18,7 +18,7 @@ Gem::Specification.new do |spec|
 
   # Specify which files should be added to the gem when it is released.
   # The `git ls-files -z` loads the files in the RubyGem that have been added into git.
-  spec.files = %w[gl_rubocop.gemspec README.md LICENSE .rubocop.yml] +
+  spec.files = %w[gl_rubocop.gemspec README.md LICENSE default.yml] +
```

Comment:
Should this also include `.rubocop.yml` ?

Reply:
Nope, we want to this to be default.yml so that rubocop's config loader knows to make the `Include` and `Exclude` paths relative to the project directory rather than the gem's directory. We're still keeping `.rubocop.yml` so that it lints the code inside this gem but it no longer has any external purpose